test/functional/wallet_basic.py

```
@@ -743,6 +745,49 @@ def run_test(self):
         txid_feeReason_four = self.nodes[2].sendmany(dummy='', amounts={address: 5}, verbose=False)
         assert_equal(self.nodes[2].gettransaction(txid_feeReason_four)['txid'], txid_feeReason_four)
 
+        self.test_chain_listunspent()
+
+    def test_chain_listunspent(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_chain_listunspent`

Add return type annotation: `None`

(ANN201)